src/main/webapp/WEB-INF/views/orders/confirm.jsp

```diff
@@ -0,0 +1,37 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Order</title>
+</head>
+<body>
+<h1>My order created</h1>
+
+<table border="1">
+<tr>
+    <th>ID</th>
+    <th>Name</th>
+    <th>Price</th>
+</tr>
+<c:forEach var="order" items="${order}">
+    <tr>
+        <td>
+            <c:out value="${order.id}"/>
```

Comment:
you are passing only products to this page, will you have orderId?

Reply:
fixed to product not order